projects/client/RabbitMQ.Client/src/client/api/AmqpTcpEndpoint.cs

```diff
@@ -182,7 +184,16 @@ public IProtocol Protocol
         /// <summary>
         /// Retrieve the SSL options for this AmqpTcpEndpoint. If not set, null is returned.
         /// </summary>
-        public SslOption Ssl { get; set; }
+        public SslOption Ssl
+        {
+            get { return _ssl; }
+            set
+            {
+                if (_ssl == null)
+                    throw new ArgumentNullException("Ssl", "Ssl property must be and instance of SslOption (cannot be null)");
```

Comment:
"An instance"

Reply:
Corrected comments in 66158d4bfb717d05f0390ae9d3670403009336fb